rclpy/src/rclpy/_rclpy_action.cpp

```diff
@@ -416,12 +400,12 @@ rclpy_action_create_client(
     error_text += action_name;
     error_text += "' : ";
     error_text += rcl_get_error_string().str;
-    rcutils_reset_error();
+    rcl_reset_error();
     throw py::value_error(error_text);
   } else if (ret != RCL_RET_OK) {
     std::string error_text{"Failed to create action client: "};
     error_text += rcl_get_error_string().str;
-    rcutils_reset_error();
+    rcl_reset_error();
     throw py::value_error(error_text);
```

Comment:
Also note these changes - it doesn't make sense to call `rcutils_reset_error()` after `rcl_get_error_string()`.